cirq-core/cirq/ops/three_qubit_gates.py

```diff
@@ -217,6 +217,55 @@ def controlled(
         )
 
 
+def decompose_all_to_all_connect_ccz_gate(
+    ccz_gate: 'CCZPowGate', qubits: Tuple['cirq.Qid', ...]
+) -> 'cirq.OP_TREE':
+    """If qubits are all-to-all connected, e.g. qubits in the same ion trap,
+    the decomposition will be:
```

Comment:
Summary line in the docstring should be a single line with <= 100 characters followed by an empty newline and then a detailed description if needed. Please update here and elsewhere

Reply:
Done. Thanks for the guide.